src/ml_tooling/plots/target_feature_distribution.py

```diff
@@ -0,0 +1,72 @@
+from matplotlib import pyplot as plt
+from matplotlib.axes import Axes
+import pandas as pd
+import numpy as np
+
+from ml_tooling.utils import DataType
+from scipy import stats
+
+
+def target_feature_distribution(
+    data: pd.DataFrame,
+    targets: DataType,
+    features: DataType,
```

Comment:
Either it should be an input DataFrame, one target DataType and a feature_name to select from the input dataframe,
or it should be an input Series for the feature to plot and an input series for the target. 

I would probably opt for the second one, as we don't allow plotting multiple features anyways